website/static/js/pages/notifications-config-page.js

```diff
@@ -18,3 +19,7 @@ $.ajax({
     $notificationsMsg.addClass('text-danger');
     $notificationsMsg.text('Could not retrieve notification settings.');
 });
+
+$(document).ready( function(){
```

Comment:
Rather than duplicating this on multiple pages, what if we check for the existence of an element that matches the `'.affix, .affix-top, .affix-bottom'` selector in `base-page.js`. If such an element exists, call `initializeResponsiveAffix()`.